Makefile

```diff
@@ -249,6 +250,22 @@ else
 SRC_DIRS := $(shell find src -type d)
 endif
 
+ifneq ($(wildcard $(EXTRACTED_DIR)/assets/audio),)
+  AIFF_EXTRACT_DIRS := $(shell find $(EXTRACTED_DIR)/assets/audio/samples -type d)
+else
+  AIFF_EXTRACT_DIRS :=
+endif
+
+ifneq ($(wildcard assets/audio/samples),)
+  AIFF_DIRS := $(shell find assets/audio/samples -type d)
+else
+  AIFF_DIRS :=
+endif
+
+AIFF_FILES         := $(foreach dir,$(AIFF_DIRS),$(wildcard $(dir)/*.wav))
```

Comment:
aren't wav and aiff different container formats? why call the variable "aiff" if the files are wav

Reply:
This was held over from when they were extracted to aiff. I've changed it to "sample" to mean any uncompressed sample container format